oshdb-util/src/main/java/org/heigit/bigspatialdata/oshdb/util/geometry/OSHDBGeometryBuilder.java

```diff
@@ -27,6 +27,7 @@
 import org.locationtech.jts.geom.Geometry;
 import org.locationtech.jts.geom.GeometryFactory;
 import org.locationtech.jts.geom.LinearRing;
+import org.locationtech.jts.geom.MultiPolygon;
```

Comment:
this import was unused, but I didn't spot this before merging :see_no_evil: sorry

Reply:
Don't worry :+1: